plugins/drill-coverage-plugin/build.gradle.kts

```diff
@@ -23,8 +22,7 @@ val adminJarDeps by configurations.creating {
 dependencies {
     commonJarDeps("org.jacoco:org.jacoco.core:$jacocoVersion")
     commonJarDeps("io.vavr:vavr-kotlin:$vavrVersion")
-
-    adminJarDeps("org.javers:javers-core:$javersVersion")
+    agentJarDeps("org.apache.bcel:bcel:6.0")
```

Comment:
remove bcel, use asm

Reply:
Disapprove! ASM is so low level for these purposes. bcel incapsulates  heavyweight logic for method body processing